### PR TITLE
339 feature keptn role readme and other readme enhancements

### LINKED ISSUE #339: [FEATURE] Keptn role documentation and Existing Readme enhancements
Keptn role readme shall be added and existing readme enhancements for the users to be easy to understand

### PR DESCRIPTION
**Pull request by eemrdog**

### CODE REVIEW

File: user-skel/ansible/roles/keptn/Readme.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# keptn
+
+This currated role can be used to install keptn on a kubernetes cluster.
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/keptn/Readme.md]: ... Keptn on a Kubernetes ... -> Capitalize

**Review by tobigremmer-dt — CHANGES_REQUESTED** [review]
Please make sure proper nouns (e.g. Kubernetes, Keptn) are capitalized. Everything else looks good.

File: user-skel/ansible/roles/keptn/Readme.md (hunk 1 above)
Comment (eemrdog) [reply user-skel/ansible/roles/keptn/Readme.md]: done

**Review by tobigremmer-dt — APPROVED** [review]
Looks good

**Pull request merged**